addons/mail/static/src/model/model_manager.js

```diff
@@ -431,25 +427,43 @@ export class ModelManager {
     }
 
     /**
+     * Adds fields, methods, and identifyingFields from the model definition to
+     * the Model. Also adds some technical variables.
```

Comment:
This doc is missing the main usage and side effect of this function, which is adding the given Model to `this.models`.
Note that maybe it should be done outside of it then?